src/components/CardGrid/CardGrid.test.tsx

```diff
@@ -9,15 +10,17 @@ describe('<CardGrid>', () => {
   it('renders and matches snapshot', () => {
     const placeholderData = generatePlaylistPlaceholder();
```

Comment:
Can you try using playlist fixture data instead of this placeholder? I think then we'll have more data filled into the snapshots like title and ID's: https://github.com/jwplayer/ott-web-app/blob/579cd0cde5d8c48933867ef3371e542252bdbbb7/src/utils/image.test.ts#L9